Intel_SighPP_CMake/Intel_SighPP_CMake/src/ml_lib/caffe_impl.cpp

```diff
@@ -25,7 +26,12 @@ struct CaffeModelImpl : public ModelInterface {
 	const float meanVal = 127.5;
 
 	const float confidence_threshold = 0.8f;
-
+	/// <summary>
+	/// Constrcutor to create a caffe-based MobileNet SSD object recognition network
+	/// </summary>
+	/// <param name="prototxt_path">path to the definition of the model architecture in a protocol buffer definition file (prototxt)</param>
```

Comment:
Maybe we should test if it works with both relative and full paths.
Currently, we use relative paths, maybe that should be specified.